doc/tutorials/duckdb-github.md

```diff
@@ -15,8 +15,11 @@ myst:
     description lang=en: Use JupySQL and DuckDB to query JSON files with SQL
     keywords: jupyter, sql, jupysql, json, duckdb
     property=og:locale: en_US
+   
 ---
 
+:og:image: https://jupysql--685.org.readthedocs.build/en/685/_static/duckdb.png
```

Comment:
in the other tutorial, you have this line in the top section (between the `---` separators). is this another way to specify the image or is this an error?

Reply:
This method does not work, since in the ref: https://sphinxext-opengraph.readthedocs.io/en/latest/#per-page-overrides we have to define tags on top of each page

Reply:
if this method doesn't work, is there any reason to keep it?

Reply:
[removed!](https://github.com/ploomber/jupysql/pull/685/files#diff-307d7b3a4330aafc19f06c2075f4b856bff94849cd29e7c89dd129eb2502817aR2)